<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>640</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>索力计数据采集分析系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MplCanvasWrapper" name="mplCanvas" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>611</height>
     </rect>
    </property>
    <widget class="QCommandLinkButton" name="cmdlnkbtn_datarfreq">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>50</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>时/频切换</string>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="cmdlnkbtn_realrhis">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>560</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>实时显示</string>
     </property>
     <property name="description">
      <string/>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>580</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_choosedate">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>580</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>选择日期</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_choosedate_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>580</y>
       <width>54</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>选择通道</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_chselect">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>580</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>通道 1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 10</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 11</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 13</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 14</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 15</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>通道 16</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_showhis">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>580</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </widget>
  </widget>
  <action name="actionFFT">
   <property name="text">
    <string>FFT设置</string>
   </property>
  </action>
  <action name="actionPath">
   <property name="text">
    <string>路径设置</string>
   </property>
  </action>
  <action name="actionSerial">
   <property name="text">
    <string>通道设置</string>
   </property>
  </action>
  <action name="actionOpenSerial">
   <property name="text">
    <string>打开串口</string>
   </property>
  </action>
  <action name="actionCloseSerial">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>关闭串口</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>建立连接</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actionNet">
   <property name="text">
    <string>网络设置</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvasWrapper</class>
   <extends>QWidget</extends>
   <header>mplcanvaswrapper.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
